config/render/capm3.yaml

```diff
@@ -837,6 +837,16 @@ spec:
         control-plane: controller-manager
     spec:
       containers:
+      - args:
```

Comment:
What happened here? Is this a generated file that changed because of a change in kustomize?

Reply:
Yes.

Reply:
OK, thanks

Reply:
Yes, that's right.